omero_fpbioimage/views.py

```diff
@@ -26,8 +49,13 @@ def fpbioimage_png(request, image_id, the_z, conn=None, **kwargs):
     """Render png for image at specified Z section."""
     image = conn.getObject('image', image_id)
     jpeg_data = image.renderJpeg(the_z, 0, compression=0.9)
-
+    width = max(image.getSizeX, image.getSizeY)
```

Comment:
You need to call these methods. ```image.getSizeX()```. This is done for you in the template, but not here!